### ISSUE TITLE
Allow to add classes on Elmish.Bulma components/elemments etc.

### ISSUE THREAD

**Issue by MangelMaxime** [MangelMaxime/Fable.Elmish.Bulma#17]
Example code:
```fs
Nav.left [ Nav.Left.classy "my-extra-class" ] [ ]
```

This would generate `"my-extra-class"` `<div class="nav-left my-extra-class"></div>`

**Comment by kunjee17** [comment]
Isn't it going in as extra html props ?

**Comment by MangelMaxime** [comment]
I will add an extra option, to all the components. because otherwise we should iterate over the `props [ .... ]` which can contain a lot of thing.

Also, by adding, `classy` and `props` (see #16 ) to all the components has will allow all the Elmish.Bulma, to have the same definition which will make the DSL consistent.
User, should never use `ClassName` on the Elmish.Bulma components.

**Comment by MangelMaxime** [comment]
Released in 0.1.0-alpha7

You can now find a `customClass` helper to append you class.

For example:

```fs
Nav.left [ Nav.Left.customClass"my-extra-class" ] [ ]
```

generate

```html
<div class="nav-left my-extra-class"></div>
```
